packages/core/core/src/loaders/plugins/index.ts

```diff
@@ -3,8 +3,8 @@ import fse from 'fs-extra';
 import { defaultsDeep, defaults, getOr, get } from 'lodash/fp';
 import { env } from '@strapi/utils';
 import type { Strapi, Plugin } from '@strapi/types';
-import { loadFile } from '../../app-configuration/load-config-file';
-import loadFiles from '../../../load/load-files';
+import { loadFile } from '../../utils/load-config-file';
```

Comment:
Why not taking this opportunity to rename this `loadConfigFile` to match the file name (makes it less confusing regarding the export just below)

Reply:
sure I can 👍

Reply:
Done